beacon_node/network/src/router.rs

```diff
@@ -611,7 +611,7 @@ impl<T: BeaconChainTypes> Router<T> {
     ) {
         let request_id = match request_id {
             AppRequestId::Sync(sync_id) => match sync_id {
-                id @ SyncRequestId::RangeBlockAndBlobs { .. } => id,
+                id @ SyncRequestId::BlocksByRange { .. } => id,
```

Comment:
Is it worth matching the `SyncRequestId` for other `_by_range_response` functions too?